### PR TITLE
Rework of the Wolfram Alpha fallback

### PR DESCRIPTION
**Pull request by penrods**
This rework does several things:
* Added ability to request sources, generating an email to the user
* Removed "did you mean", which just gave interpretations you never meant :)
* Removed repetition of the question.  It was awkward and stilted experience, at best.
* FIX: Changed normalization for special cases -- "how many inches in meter" wouldn't handle as well as "how many inches in a meter".
* Changed to Apache 2.0 licensing
* Switch the private API key storage to skill settings instead of mycroft.conf (with migration code)
* Switched to self.log instead of LOG
* Commented code!

### CODE REVIEW

File: __init__.py [hunk 1]
```diff
@@ -1,38 +1,30 @@
...
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 #
-# You should have received a copy of the GNU General Public License
-# along with Mycroft Core.  If not, see <http://www.gnu.org/licenses/>.
-
-
 from StringIO import StringIO
```
Comment (forslund) [inline __init__.py:15]: Not critical but to make this skill run under both python 3 and python 2 

```python
from io import BytesIO
```

should be used

**Review by LearnedVector — CHANGES_REQUESTED** [review]
we should start adding a .gitignore to these skills like __init__.pyc and settings.json

File: __init__.py (hunk 1 above)
Comment (penrods) [reply __init__.py:15]: I'll do that in a future update.

**Comment by penrods** [comment]
@Mn0491 added .gitignore

**Review by LearnedVector — APPROVED** [review]

**Pull request merged**